etc/org.opencastproject.editor.EditorServiceImpl.cfg

```diff
@@ -94,3 +94,15 @@
 # This Property sets the time (in seconds) after which the editor should refresh the lock on a mediapackage.
 # Default: 60
 #lock.refresh.after.seconds=60
+
+# Default values for  trim segments in the editor.
+# Default: In milliseconds
+#
+# This property sets the length of the default trim segment at the beginning and end of each video
+# Default: 3000
+default.padding.length = 3000
+
+# This property sets the minimum duration of the video to add default trim segments to. If the video is shorter than
+# this duration, no default trimm segment will be added to it.
+# Default: 6000
+min.video.duration = 6000
```

Comment:
The default configuration should not add trim segments. Also please comment out default configuration keys.